app/serializers/preprint.js

```diff
@@ -1,4 +1,19 @@
 import OsfSerializer from 'ember-osf/serializers/osf-serializer';
 
 export default OsfSerializer.extend({
+    serialize(snapshot) {
+        // Normal OSF serializer strips out relationships. We need to add back primaryFile for this endpoint
+        let res = this._super(...arguments);
+        res.data.relationships = {
+            // Not sure what the name of this key comes from, but it's required.
```

Comment:
Thanks guys, let's merge this in and we'll continue work as things need to be changed.